### PR TITLE
ci: add option to force install from upstream repo

### PR DESCRIPTION
**Pull request by mudler**
Bump to latest luet for bootstrap

Signed-off-by: Ettore Di Giacinto <edigiacinto@suse.com>

### CODE REVIEW

**Comment by mudler** [comment]
@Itxaka I could also reuse this branch to directly publish to arm64 bypassing tests if we make the build work, wdyt?

File: .github/build.yaml.gomplate [hunk 1]
```diff
@@ -72,8 +72,19 @@
...
       - name: Run make {{{ $target }}}
         run: |
-          {{{ if eq $target "deps_ci" }}}export DOCKER_INSTALL=true{{{end}}}
+          {{{ if eq $target "deps_ci" }}}
+          export DOCKER_INSTALL=true
+          {{{end}}}
+          {{{ if ne $config.luet_script_override "" }}}
+          export DOCKER_INSTALL=false
+          export LUET_INSTALL_FROM_COS_REPO=false
+          export LUET_INSTALL_FROM_COS_REPO=false
```
Comment (Itxaka) [inline .github/build.yaml.gomplate:81]: duplicated entry

File: .github/config/nightly.yaml [hunk 2]
```diff
@@ -24,6 +24,7 @@ flavors:
         build_image_qemu: true
         build_image_vbox: true
         build_raw_image: true
+        luet_script_override: ""
```
Comment (Itxaka) [inline .github/config/nightly.yaml:27]: so the value is false or empty?

Comment (mudler) [reply .github/config/nightly.yaml:27]: I just changed it slightly, it is a string that you can use to specify a luet version to use directly :+1:

File: .github/workflows/build-master-blue-arm64.yaml [hunk 3]
```diff
@@ -62,10 +62,18 @@ jobs:
       - name: Run make deps_ci
         run: |
           export DOCKER_INSTALL=true
+          export DOCKER_INSTALL=false
```
Comment (Itxaka) [inline .github/workflows/build-master-blue-arm64.yaml:65]: duplicated entry here too int he final output

Comment (mudler) [reply .github/workflows/build-master-blue-arm64.yaml:65]: argh there it's a bit not nice from the template itself, there the first one is coming from another option, not sure if want to make it too tight to the override

File: .github/workflows/build-master-blue-arm64.yaml [hunk 4]
```diff
@@ -62,10 +62,18 @@ jobs:
...
+          export LUET_INSTALL_FROM_COS_REPO=false
+          export LUET_VERSION=0.32.0
           sudo -E make deps_ci
```
Comment (Itxaka) [inline .github/workflows/build-master-blue-arm64.yaml:68]: shouldnt deps_ci call the get_luet script? Like it does it already no?

Comment (mudler) [reply .github/workflows/build-master-blue-arm64.yaml:68]: yup it does, but then when we call make deps we overwrite it with the version coming from the repos.. it's not nice, we re-install it twice here

Comment (mudler) [reply .github/workflows/build-master-blue-arm64.yaml]: reworked it slightly, at least now should be a bit more obvious

Comment (mudler) [reply .github/workflows/build-master-blue-arm64.yaml]: and I broke it somehow :D

Comment (mudler) [reply .github/workflows/build-master-blue-arm64.yaml:65]: oh I see, ci is calling `deps_ci` now. adapted it

**Comment by mudler** [comment]
@Itxaka arm64 repo were published, adapted to drop the overrides but keep the handles in the pipeline config (if we might need that in future)

**Pull request merged**